src/apps/pulse/hooks/useIntentSdk.ts

```diff
@@ -0,0 +1,39 @@
+import { IntentSdk, Options } from '@etherspot/intent-sdk';
+import { useWalletAddress } from '@etherspot/transaction-kit';
+import { useEffect, useState } from 'react';
+import { createWalletClient, custom, Hex } from 'viem';
+import { usePrivy, useWallets } from '@privy-io/react-auth';
```

Comment:
Just a note - we will eventually move away from Privy. Nothing to action now, just a note